Source/WebKit/UIProcess/API/ios/WKWebViewTestingIOS.mm

```diff
@@ -330,6 +330,9 @@ - (NSString *)_scrollbarState:(unsigned long long)rawScrollingNodeID processID:(
         {
             TextStream::GroupScope scope(ts);
             ts << ([_scrollView showsHorizontalScrollIndicator] ? ""_s : "none"_s);
+#if HAVE(UIKIT_SCROLLBAR_COLOR_SPI)
+            ts << ([_scrollView _verticalScrollIndicatorColor] ? WebCore::colorFromCocoaColor([_scrollView _verticalScrollIndicatorColor]).debugDescription() : ""_s);
+#endif
```

Comment:
I think we're missing the equivalent logging change here? To include horizontal when necessary.

Reply:
Fixed.